test/libsolidity/ASTJSON/fallback_and_reveice_ether.json

```diff
@@ -16,6 +16,7 @@
       "baseContracts": [],
       "contractDependencies": [],
       "contractKind": "contract",
+      "emittedEvents": [],
```

Comment:
Typo in test name: `fallback_and_reveice_ether.json` -> `fallback_and_receive_ether.json`